### DISCUSSION TITLE
LISEM development Discussion

### DISCUSSION THREAD

**Discussion by bastianvandenbout** [bastianvandenbout/LISEM#7] [Ideas]
Dear Users,

I was wondering if there were some people interested to host a small discussion on the LISEM development.
Some video(optional) chat where some feature requests, bug reports etc.. could be discussed.
As well as trouble-shooting for users, and feedback on the current version.
This would be hosted on discord probably. If sufficient people like this announcement I will post here about a moment for this.

Kind regards,
bastian

**Comment by bastianvandenbout** [comment]
Okey, so we'll see how many people join, but I will try to make a regular moment available. 
As of now (2-11-2022): I will make a discussion/Q&A moment on the Discord server.
To accomodate the different time-zones of the users, I will be there for a two-hour period, between 13:00 and 15:00 GMT+1 (Netherlands Time).
I'll try to find a moment every month or so. For now:
Here you can find the discord server:  https://discord.gg/7AuXufd65F 

Moments: 
- Tuesday the 8th of november between 13:00 and 15:00

Previous moments:
None

**Comment by lozaayehu** [comment]
Dear Prof,
I would like to share with you the error I encountered while running my data for a Simple slope stability analysis. I constantly have this error message on the Console box. Could you please guide me through this? I copy and past the error, I constantly have Thank you so much for your help.

"compiling C:/Program Files/lisem/bin/kernels/model_flow.cl flow
compilation error clBuildProgram(õ) CL_BUILD_PROGRAM_FAILURE
Build log for gfx902: C:\Users\acer\AppData\Local\Temp\OCL17332T1.cl:247:37: error: read_image expects readable image argument float rain = read_imagef(RAIN,sampler, int2(gx_x1,gy)).x; ~~~~~~~~~~~^ 1 error generated. error: Clang front-end compilation failed! Frontend phase failed compilation. Error: Compiling CL to IR
CL ERROR, QUITTING MODEL!
Deleting all model-related memory objects
Finished deleting all model related memory objects
Do Model Run on thread"

**Comment by Engqaline** [comment]
Hi, openlisem developer, i'm new for this software and i want to download and install the software. as i saw for in version 0.2.4 [Windows-64x Portable zipped executable], [Windows-64x installer], and [Windows-64x Python Wheel]. Also there is two other versions 0.2.3 and 0.2.2. so please which one i use. further, if there is video for how to install please share me. 
thanks
